includes/blocks.inc

```diff
@@ -106,6 +106,40 @@ function islandora_solr_block_info() {
 }
 
 /**
+ * Implements hook_block_configure().
+ */
+function islandora_solr_block_configure($delta = '') {
+  if ($delta == 'simple') {
+    $form = array();
+    $form['islandora_solr_simple_search_label_title'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Simple Search Label'),
+      '#size' => 30,
+      '#default_value' => variable_get('islandora_solr_simple_search_label_title', 'Search Term'),
+      '#description' => t('Customize label text associated with search input form. Accessibility guidelines recommend whenever possible to explicitly associate label text with form inputs.'),
```

Comment:
@dmcollins Just wondering about the "Accessibility guidelines...." note. The form requires you to enter a label and the generated form properly associates the label as a `<label for="simple-search-textfield">`. So while informative I'm not clear on its purpose. What are you trying to accomplish with that text?